build/Targets/VSL.Imports.targets

```diff
@@ -413,15 +423,16 @@
 
     <!-- The below check is to decide if the assembly produced by the VSIX project should be included in the VSIX-->
     <ItemGroup>
-      <VsixInputAssembliesToSign Include="@(IntermediateAssembly)" Condition="'$(IncludeAssemblyInVSIXContainer)' == 'true'" />
+      <VsixInputAssembliesToSign Include="@(IntermediateAssembly)" Condition="'$(IncludeAssemblyInVSIXContainer)' == 'true'">
+        <StrongName>72</StrongName>
```

Comment:
Currently `ProjectSystem.dll` is not set with the property `SignAssembly=true`. Hence `ProjectSystem.dll` will not enter the condition which assigns `GetTargetPathWithStrongNameFriendlyID`. Will introduce a condition